pkg/api/common/types.go

```diff
@@ -0,0 +1,55 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+package common
+
+type ContainerdConfig struct {
+	OomScore  int     `toml:"oom_score,omitempty"`
+	Root      string  `toml:"root,omitempty"`
+	Subreaper bool    `toml:"subreaper"`
```

Comment:
Please remove this, it's not a real config. I removed it from the containerd docs: containerd/containerd#4194

Reply:
Same for the actual generated toml.

Reply:
gah, thought I pulled this out. will remove